CHANGELOG.md

```diff
@@ -17,7 +17,11 @@ x.x.x Release notes (yyyy-MM-dd)
 
 ### Bugfixes
 
-* None.
+* Fix a call to `commitWrite(withoutNotifying:)` committing a transaction that
+  would not have triggered a notification incorrectly skipping the next
+  notification.
+* Fix collection notifications missing move information for non-end insertions
```

Comment:
The bug that was fixed was when sync merges produced a non-end insertion of a row into a table, not anything to do with RLMArray.